wiki/development/coding-guidelines.md

```diff
@@ -8,27 +8,185 @@ order: 1
 ---
 
 
-## 1. Indentation
+## 1. Naming Conventions
 
-4 spaces for indentation.
+### 1.1. Variable Names
+#### 1.1.1. Global Variable naming
+All global variables must start with the ACE prefix followed by the component, separated by underscores. Global variables may not contain the fnc_ prefix if the value is not callable code.
```

Comment:
`ACE` ... `fnc`
or (whichever is selected should be used on the entire page)
`ACE_` ... `fnc_`